sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -129,6 +130,15 @@ def _set_partition_key(
             return _return_undefined_or_empty_partition_key(self.is_system_key)
         return cast(Union[str, int, float, bool, List[Union[str, int, float, bool]]], partition_key)
 
+    def _get_epk_range_for_partition_key(
+            self,
+            partition_key_value: Union[str, int, float, bool, Sequence[Union[str, int, float, bool, None]], Type[NonePartitionKeyValue]]) -> Range: # pylint: disable=line-too-long
```

Comment:
For the partition_key_value type, couldn't you use the `PartitionKeyType` defined above.

Reply:
good catch, yes I should, will update in next iteration

Reply:
updated